Makefile

```diff
@@ -320,8 +320,8 @@ gocyclo:
 govet:
 	go vet $(shell go list ./agent/... | grep -v /testutils/ | grep -v _test\.go$ | grep -v /mocks | grep -v /model)
 
-GOFMTSTRING:='{{$$dir := .Dir}}{{range $$f := .GoFiles}}{{$$dir}}/{{$$f}} {{end}}{{range $$f := .IgnoredGoFiles}}{{$$dir}}/{{$$f}} {{end}}'
-GOFMTFILES:=$(shell go list -f $(GOFMTSTRING) ./agent/...)
+GOFMTSTRING:='{{$$dir := .Dir}}{{range $$f := .GoFiles}}{{$$dir}}/{{$$f}} {{end}}{{range $$f := .IgnoredGoFiles}}{{$$dir}}/{{$$f}} {{end}}{{range $$f := .TestGoFiles}}{{$$dir}}/{{$$f}} {{end}}'
+GOFMTFILES:=$(shell go list -tags functional -f $(GOFMTSTRING) ./agent/...)
```

Comment:
won't this ignore test files that aren't tagged with 'functional' ?

Reply:
The files that aren't tagged with 'functional' will not be ignored.

Reply:
This will not ignore the test files that aren't tagged 'functional'. Adding the tag, we will filter out more files that tagged 'functional' are missing in the previous GOFMTFILES.